indexing-solr/src/main/java/edu/unc/lib/boxc/indexing/solr/filter/SetDescriptiveMetadataFilter.java

```diff
@@ -439,4 +444,10 @@ private String formatName(Element nameEl) {
         }
         return nameValue;
     }
+
+    private String setDateYear(Date date) {
```

Comment:
Just due to java naming conventions I'd prefer not to use "set" here, could you name this something like "extractDateYear"